pkg/utils/addons/addons.go

```diff
@@ -0,0 +1,215 @@
+package addons
+
+import (
+	mf "github.com/manifestival/manifestival"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type spec struct {
+	runtime          string
+	version          string
+	minorVersion     string
+	supportedVersion string
+}
+
+var (
+	runtimeLabel                = "pipeline.openshift.io/runtime"
+	typeLabel                   = "pipeline.openshift.io/type"
+	supportedVersionsAnnotation = "pipeline.openshift.io/supported-versions"
+	preserveNSAnnotation        = "operator.tekton.dev/preserve-namespace"
```

Comment:
some of these are already there in `pkg/flag.go`. 
Let us reuse or move the additional ones into `pkg/flag/flag.go`

Reply:
Thank you, will update it.